### PR TITLE
Updating MFTF info for 2.3.13 release.

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

Update docs for the MFTF 2.3.13 release.

whatsnew
Added two new [PWA actions](http://devdocs.magento.com/mftf/2.3/test/actions.html#waitforpwaelementnotvisibletype) for the MFTF 2.3.13 release.

### CODE REVIEW

File: _data/codebase/v2_3/open-source/composer_lock.json [hunk 1]
```diff
@@ -6345,7 +6345,7 @@
         },
         {
             "name": "magento/magento2-functional-testing-framework",
-            "version": "2.3.9",
+            "version": "2.3.13",
```
Comment (dshevtsov) [inline _data/codebase/v2_3/open-source/composer_lock.json]: do not edit codebase files manually
they can be copied from the source codebase entirely with no changes only

File: guides/v2.1/architecture/archi_perspectives/framework.md [hunk 2]
```diff
@@ -33,13 +33,15 @@ The Framework is responsible for operations that are useful for potentially all
 Here is the Magento Framework folder structure:
 
 
-`vendor/
+```bash
```
Comment (dshevtsov) [inline guides/v2.1/architecture/archi_perspectives/framework.md]: file tree doesn't relate to Bash
change to `tree` or `terminal`

File: guides/v2.1/architecture/archi_perspectives/framework.md [hunk 3]
```diff
@@ -54,120 +56,29 @@ The `vendor/magento/framework` directory maps to the `Magento\Framework` {% glos
...
-      </tr><tr>
-         <td><code>Magento\Framework\Data</code></td>
-         <td>Contains additional classes that handle forms.</td>
-      </tr><tr>
-         <td><code>Magento\Framework\Url</code></td>
-         <td>Contains code to look up other pages in Magento.</td>
-      </tr>
-   </tbody>
-</table>
+|Namespace|Purpose|
```
Comment (dshevtsov) [inline guides/v2.1/architecture/archi_perspectives/framework.md]: wrap each namespace in back-ticks like `Magento\Framework\DataObject`

File: guides/v2.1/architecture/archi_perspectives/framework.md [hunk 4]
```diff
@@ -54,120 +56,29 @@ The `vendor/magento/framework` directory maps to the `Magento\Framework` {% glos
...
-		<tr>
-			<td>
-				<code>Magento\Framework\Validator</code>
-			</td>
-			<td>Contains the code that validates data (currencies, not empty) and that handles observers for any Magento event.
-			</td>
-		</tr>
-	</tbody>
-</table>
+|Namespace|Purpose|
```
Comment (dshevtsov) [inline guides/v2.1/architecture/archi_perspectives/framework.md]: wrap each namespace in back-ticks

**Comment by dshevtsov** [comment]
@dobooth 
1. Change the PR title to more verbose. It have to be clear to any contributor.
2. In whatsnew, change '2' to 'two'. For more details, refer to the internal Style guide > Grammar > Numbers

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: Updated but removed file from this PR.

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: Done.

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 4 above)
Comment (dobooth) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: Done.

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 3 above)
Comment (dshevtsov) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: I don't see the changes

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 4 above)
Comment (dshevtsov) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: I don't see the changes

File: guides/v2.1/architecture/archi_perspectives/framework.md (hunk 2 above)
Comment (dshevtsov) [reply guides/v2.1/architecture/archi_perspectives/framework.md]: Nothing changed

**Comment by dobooth** [comment]
What a mess.

**Pull request closed without merging**